### ISSUE TITLE
Item "Get iostats" in ZFS pool discovery fix

### ISSUE THREAD

**Issue by cedric-marcoux** [Cosium/zabbix_zfs-on-linux#42]
Hello,

the item "Zpool {#POOLNAME}: Get iostats" in ZFS discovery pool value should be changed from
vfs.file.contents[/proc/spl/kstat/zfs/{#POOLNAME}/io]
to
vfs.file.contents[/proc/spl/kstat/zfs/{#POOLNAME}/iostats]

Regards

**Comment by kobuki** [comment]
This fix doesn't work in itself. I had to change the base item key to
```
vfs.file.regexp[/proc/spl/kstat/zfs/rpool/iostats,"^[0-9]"]
```
to only return the first line of the file. Then the dependent items work, but always show zero IO operations. Something must have changed in the format of the stats file. Using Zabbix 5.0.27.

**Comment by tuxick** [comment]
> 

Yes, that first line is/became rather useless. `zpool iostat ` is the way.
